bind/gencffi_func.go

```diff
@@ -109,6 +109,36 @@ func (g *cffiGen) genFuncBody(f Func) {
 
 	g.wrapper.Printf("_cffi_helper.lib.cgo_func_%[1]s(%[2]s)\n", f.id, strings.Join(funcArgs, ", "))
 
+	if f.err {
+		switch nres {
+		case 1:
+			g.wrapper.Printf("if not _cffi_helper.lib._cgopy_ErrorIsNil(cret):\n")
+			g.wrapper.Indent()
+			g.wrapper.Printf("c_err_str = _cffi_helper.lib._cgopy_ErrorString(cret)\n")
```

Comment:
I haven't checked, but, do you know what's the memory ownership model for C-strings being passed by `_cgopy_ErrorString` ?
and what is its interaction with `ffi.string` ?

ie: who's supposed to `free` the C-string ?